bundles/af-core/src/test/resources/schema/0.11.0-Pre/adaptive-form.schema.json

```diff
@@ -1038,6 +1082,9 @@
             {
               "$ref": "#/$defs/Image"
             },
+            {
```

Comment:
This is already added as part of 0.12.0. you can remove these changes

Reply:
Okay.